auraescript/src/cri.rs

```diff
@@ -30,7 +30,7 @@
 
 // TODO: macro doesn't support streaming. Does deno?
 macros::ops_generator!(
-    kubernetes::cri,
+    kubernetes::cri("gen/kubernetes/cri/v1/release-1.26.ts"),
```

Comment:
these all start with gen and you add "gen" to the out_dir above. do we get double-gen?

Reply:
We don't get a double "gen" as I took that out of the macro logic for locating the ts. I dropped it in hopes that I could have docs with examples, which I got to, but then realized the macro outputs files, and the example would also output files when testing as cargo tests rust code in docs too, so I dropped the docs example.

In other words, I can put the assumption back into the macro if that is preferred?

Reply:
i think it's a bit easier than trying to remember to put "gen" in front every time. but it's a small thing.